cypress/afterScreenshot.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * This is a hook that runs after a screenshot is taken using Cypress.
+ * We use this hook to do a couple of tasks.
+ *
+ * @author Liam Mulhall <liammulh@gmail.com>
+ */
+
+// This needs a CommonJS require because it's a CommonJS module.
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const pixelmatch = require('pixelmatch')
+import {PNG} from 'pngjs'
+import {readFileSync, readdirSync, rmSync} from 'node:fs'
+import {resolve} from 'node:path'
+
+const SCREENSHOT_DIR = resolve(process.cwd(), 'cypress', 'screenshots')
+
+/**
+ * Verifies that two screenshots are close.
+ */
+const screenshotsAreClose = (options: {
```

Comment:
Rereading this, I think `actualPath` and `expectedPath` should be passed explicitly rather than passed in the options object.

Reply:
Done in https://github.com/numberscope/frontscope/pull/245/commits/da9377bf7cac1f377b45ba8b2cafb8e1a077c5a7.